src/scmrepo/git/lfs/client.py

```diff
@@ -272,3 +275,18 @@ def _as_atomic(to_info: str, create_parents: bool = False) -> Iterator[str]:
         raise
     else:
         shutil.move(tmp_file.name, to_info)
+
+
+class _ExponentialRetry(ExponentialRetry):
+    def get_timeout(
+        self, attempt: int, response: Optional[aiohttp.ClientResponse] = None
+    ) -> float:
+        if response is not None and response.status == HTTPStatus.TOO_MANY_REQUESTS:
+            if "Retry-After" in response.headers:
+                with suppress(ValueError):
+                    return float(response.headers["Retry-After"])
+            for k in ["RateLimit-Reset", "X-RateLimit-Reset"]:
+                if k in response.headers:
+                    with suppress(ValueError):
+                        return float(response.headers[k]) - time()
```

Comment:
just to make sure - is it always UTC in headers? 

it might also happen in rare cases that we get a negative number here - need to check the code above is ready for that

Reply:
Both gitlab/github return Unix time.

Reply:
I don't think we need to worry about negative times. Even if it does, it won't wait for that time, and should be fine.

Reply:
sounds good, folks!